Docs: VAULT_DISABLE_REDIRECTS added further clarity (#27256)
hashicorp/vault: website/content/docs/commands/index.mdx

### `VAULT_DISABLE_REDIRECTS`

> Prevents the Vault client from following HTTP redirects. By default, the Vault CLI client will only follow a single redirect and any additional redirection can result in an empty or unexpected response. 

Reviewer: ```suggestion
Prevents the Vault client from following HTTP redirects. By default, the
Vault CLI client only follows the first redirect. Additional redirects can
result in an empty or unexpected response. 
```
Style correction: write in active voice
